src/ensembl/src/content/app/species/state/general/speciesGeneralHelper.ts

```diff
@@ -569,33 +569,42 @@ const getExampleLinks = (props: {
       : undefined;
 
     if (focusId) {
-      exampleLinks.genomeBrowser = urlFor.browser({
-        genomeId: genome_id,
-        focus: focusId
-      });
-
-      exampleLinks.entityViewer = urlFor.entityViewer({
-        genomeId: genome_id,
-        entityId: focusId
-      });
-    }
-  } else if (section === SpeciesStatsSection.ASSEMBLY) {
-    const regionExample = exampleFocusObjects.find(
-      (object) => object.type === 'region'
-    );
-
-    const focusId = regionExample?.id
-      ? buildFocusIdForUrl({
-          type: 'region',
-          objectId: regionExample.id
-        })
-      : undefined;
+      exampleLinks = {
+        genomeBrowser: {
+          url: urlFor.browser({
+            genomeId: genome_id,
+            focus: focusId
+          })
+        },
+        entityViewer: {
+          url: urlFor.entityViewer({
+            genomeId: genome_id,
+            entityId: focusId
+          })
+        }
+      };
+    } else if (section === SpeciesStatsSection.ASSEMBLY) {
```

Comment:
There's something wrong with the nesting here — this condition ended up inside the `SpeciesStatsSection.CODING_STATS` block.

<img src="https://user-images.githubusercontent.com/6834224/108290665-c9333780-7188-11eb-81c6-df36506401ef.png" width="600px" />

I noticed it only accidentally when running the type checking script 😞 

![image](https://user-images.githubusercontent.com/6834224/108290577-99842f80-7188-11eb-8e69-77446055a236.png)

Reply:
Thanks for spotting that. I'll look into it now.

Reply:
That's fixed now ✅